Essentials/src/com/earth2me/essentials/api/Economy.java

```diff
@@ -170,17 +238,64 @@ public static void add(String name, double amount) throws UserDoesNotExistExcept
         }
     }
 
+    /**
+     * Adds money to the balance of a user
+     *
+     * @deprecated Usernames can change, use {@link Economy#add(UUID, BigDecimal)} or {@link Economy#add(User, BigDecimal)}
+     *
+     * @param name   Name of the user
+     * @param amount The money you want to add
```

Comment:
Consider rewording all around to something more like 'The amount of money to be added to the user's account'